python/ml4ir/applications/ranking/model/ranking_model.py

```diff
@@ -280,6 +281,19 @@ def mask_padded_records(predictions, features_dict):
             pad_sequence=pad_sequence,
         )
 
+        # Logging positional biases
+        for layer in self.model.layers:
+            if layer.name == PositionalBiasHandler.FIXED_ADDITIVE_POSITIONAL_BIAS:
+                positional_bias_coefficients = pd.DataFrame(
+                    [{'rank': i + 1, 'positional_bias': layer.get_weights()[0][i][0]}
+                     for i in range(len(layer.get_weights()[0]))]
+                )
+                self.file_io.write_df(
```

Comment:
@mohazahran Might be worth passing them through a `scipy.special.softmax` as well, no?

Reply:
I think using the raw score would help better in reconstructing the scores for training if needed.

Reply:
Log both. Softmax is easier to interpret.